src/api/entities/Identity/index.ts

```diff
@@ -891,6 +893,37 @@ export class Identity extends Entity<UniqueIdentifiers, string> {
     return assembleResult(didRecords);
   }
 
+  /**
+   * Retrieve all assets with given permissions where this Identity is an Agent
```

Comment:
Retrieve all the Security Tokens for which this Identity is an Agent, with the corresponding permission group